tests/python/unittest/test_exc_handling.py

```diff
@@ -165,6 +166,46 @@ def test_multiple_waitalls():
     assert caught, "No exception thrown"
     mx.nd.waitall()
 
+@with_seed()
+def test_exc_profiler():
```

Comment:
seems a bit heavy for a unittest, I think passing some nd.ones() through a simple dense layer would do the same right?

Reply:
I have simplified it. Please take a look